README.md

```diff
@@ -492,6 +492,21 @@ java -jar $DJINNI_JAR_DIR/djinni-assembly-0.1-SNAPSHOT.jar \
 
 *Note*: The `all` target of the main `Makefile` includes the `djinni_jar` target.
 
+## Generate an apple universal binary
+
+The `mac-build-support-lib.sh` helps you to build an universal static library.
+It uses the platform file of the [ios-cmake](https://github.com/leetal/ios-cmake) repository.
+
+It basically creates one universal static library per `IOS_PLATFORM` variable and uses `lipo` 
```

Comment:
The combination of "Mac" in the script name and "IOS" here makes me wonder how general this feature is.  Does it cover all Apple platforms, or just iOS?

Reply:
The name is bad, I have to admin. The generated binaries are for iOS platform, but to be compiled from a mac. I will rename the file.